packages/@atjson/source-html/src/converter/third-party-embeds.ts

```diff
@@ -123,5 +134,39 @@ export default function convertThirdPartyEmbeds(doc: Document) {
       doc.removeAnnotations(scripts);
     });
 
+  /**
+   * CNE Audio script code
+   * The script code has this format:
+   *
+   *  <script src="https://{host}/script/{type}/{id}?skin={brand}&target={div_id}" defer></script><div id="{div_id}"></div>
+   *
+   * The Iframe code has this format:
+   *
+   *  <iframe src="https://{host}/iframe/{type}/{id}?skin={brand}" frameborder="0" height="244" sandbox=allow-scripts allow-popups allow-popups-to-escape-sandbox"></iframe>
+   *
+   */
+  doc
+    .where(isCneAudioScript)
+    .as("embed")
+    .update((embed) => {
+      const anchorName = `js-audio1-${new Date().getTime()}`;
```

Comment:
I don't think this should be generated here. The `ID` attribute we assign to the div is a render-time concern. The `anchorName` on the annotation should be an editorially-defined attribute which corresponds to a part of a page someone can link to via the url hash (like `https://www.your-page.com#audio-player`)

Reply:
To disambiguate, I would suggest that anchor is parsed from a different ID
ex
```
<script id="audio-player" src="https://{host}/script/{type}/{id}?skin={brand}&target={div_id}" defer></script><div id="{div_id}"></div>
```

Reply:
Or we can do what Tim added above in the renderer where we have an extra wrapping div